src/main/webapp/package.json

```diff
@@ -70,7 +71,7 @@
     "react-markdown": "^8.0.2",
     "react-mde": "^11.5.0",
     "react-redux": "^7.2.8",
-    "react-router-dom": "^6.3.0",
+    "react-router-dom": "^6.4.0-pre.7",
```

Comment:
Is there a specific feature you are using from the pre release?

Reply:
Two major features that are about to come out in the next update that they have taken from their other project Remix.

1.  [loader](https://beta.reactrouter.com/en/dev/route/loader) better way of loading data onto the component then calling it within a useEffect hook.
2.  [errorElement](https://beta.reactrouter.com/en/dev/route/error-element): provides fallback UI incase their is an error loading the components (whether an ajax error or otherwise)